packages/tailwind-utils/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "src",
+    "outDir": "dist"
+  },
+  "include": ["src/**/*"],
+  "exclude": ["node_modules", "dist"]
```

Comment:
You shouldn't need to exclude these unless you have a broad scope in include.

Reply:
The other option would be to keep exclude and let include default to `**/*`, but that doesn't make it any thiccer